examples/p1.4.1/tenant/external-upgrade.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: hcp.hpe.com/v1
+kind: Tenant
+metadata:
+  name: tenant-1
+spec:
+  baseimagetag:  "202112011100P150"
+  clustername:   dataplatform
+  clustertype:   external
+  corelocation:  /var/lib/docker/mapr/cores
+  grouplist:
+    - hpecp-tenant-7-member
```

Comment:
Please add the comment for command to fetch the grouplist.

Reply:
Sure. Thanks for reminding me.